### ISSUE TITLE
Email to all lists to announce event

### ISSUE THREAD

**Issue by ashumz** [ashumz/citycamp#13]

**Comment by ashumz** [comment]
which lists? Are the ones listed at the bottom of https://docs.google.com/a/rallydev.com/document/d/1gSDShZdnP8cdaxYfKEo5fsRbh1J0PeWb6n7EHdNUJ4c/edit accurate?

**Comment by jhibbets** [comment]
Do you still need help with this? (Still catching up on email from last
week.)

**Comment by ashumz** [comment]
Yeah @jhibbets -- could use a little guidance. I need to find out if those are the ones I should use/if everything is up to date as far as you know, and if I should send the same update that I sent to the Google Group? 

Text (would update it with the keynote speaker info as well as the next meetup event info):

See You @ CityCampNC 2015
Hi! I'm Ashley, head of the marketing committee and volunteer for this year's CityCampNC event. We're excited to announce that this year's event will take place June 11-13 in Raleigh. 

Call for Sponsors 
CityCampNC is 100% run by volunteers and funded by our sponsors. We're in the process of recruiting sponsors for this year's event. Our 2015 sponsorship opportunities are detailed on the CityCampNC website here. If you have any leads on sponsorship opportunities, please get in touch with the head of our sponsorship committee and CityCampNC organizer, Chris Mathews.

Call for Speakers
We have booked our keynote speaker for Friday's event, and we are actively looking for speakers for Thursday night's lightning talks and Friday. If you're interested in speaking, or know someone who might be, please contact CityCampNC organizer, Jason Hibbets. 

CityCamp Event Information
This year’s event will be at two locations. We are still confirming a location for our lightning talks/“Taste of CityCamp” on Thursday evening, June 11. Friday and
Saturday's (June 12-13) CityCamp events will take place at the Wake County Commons building. More info on schedule will be coming soon!

We'll announce our keynote speaker for the event soon - stay tuned! Information will continue to be made available as we continue planning the event at http://citycampnc.org. If you haven't already, follow @citycampnc on Twitter for regular updates.

Interesting in Helping Out with CityCampNC 2015? 
We have an awesome group of volunteers for this year's event so far -- and we'd love your help, too. We're meeting every 2 weeks currently, with our next virtual meeting on April 13 at 6:30 pm (Meeting info located here.)

Let's make this year's CityCampNC the best one yet. :)

**Comment by jhibbets** [comment]
I started going through this and didn't realize it was that extensive. I'll
need more time.

Here is an updated blurb I sent to the CAC folks:

CityCamp NC Event Information

CityCamp NC is three days of open sourced talks, workshops, and hands-on
problem solving, to re-imagine the way the web, applications, technology,
and participation will shape the future of NC cities. CityCamp brings
together government, business, neighborhood, non-profit, and academic
communities to work toward next generation solutions for local
municipalities.

This year’s event will be held on June 11-13. On Thursday, June 11, join us
for lightning talks and the “Taste of CityCamp." The conference continues
on Friday and
Saturday, June 12-13 with a GIS panel, the keynote presentation,
unconference workshops, and hackathon.

Information will continue to be made available as we continue planning the
event at http://citycampnc.org. If you haven't already, follow @citycampnc
on Twitter for regular updates.
Interesting in Helping Out with CityCampNC 2015?
We have an awesome group of volunteers for this year's event so far -- and
we'd love your help, too. Find our next cityCamp NC meeting at:
http://www.meetup.com/Triangle-Code-for-America/

**Comment by ashumz** [comment]
This is what I used for the Mailchimp draft: 

Subject:  Save the Date! CityCampNC 2015 is June 11-13!

Body:
**CityCampNC 2015 is June 11-13!**

We're excited to announce that this year's event will take place June 11-13 in Raleigh.

CityCamp NC is three days of open sourced talks, workshops, and hands-on problem solving, to re-imagine the way the web, applications, technology, and participation will shape the future of NC cities. CityCamp brings together government, business, neighborhood, non-profit, and academic communities to work toward next generation solutions for local municipalities.

**Event Schedule and Location**

On Thursday, June 11, join us for lightning talks and the “Taste of CityCamp” at HQ Raleigh.

The conference continues at the Wake County Commons building on Friday and Saturday, June 12-13 with a GIS panel, the keynote presentation, unconference workshops, and hackathon. Information will continue to be made available as we continue planning the event at http://citycampnc.org (http://citycampnc.org/)

Registration will open within the next few weeks, so stay tuned!

**Interesting in Helping Out with CityCampNC 2015?**

We have an awesome group of volunteers for this year's event so far, and we'd love your help, too. We're holding a virtual meeting every Monday until the event. You can RSVP for the meetings at http://www.meetup.com/Triangle-Code-for-America/ where all of the CityCamp NC meetings are posted.

We look forward to seeing you there! If you have questions, please reply to this email. If you haven't already, you can also follow @citycampnc (http://twitter.com/citycampnc) on Twitter for regular updates.

See you there,

Ashley & The CityCampNC Team

**Comment by ashumz** [comment]
@jhibbets we should ask at tonight's meeting if anyone can send this out to groups they are involved with - I could join all of those mailing lists and post it, but it'd probably be more effective to see if we can find people with direct ties to some of the groups

**Comment by ashumz** [comment]
At tonight's meeting, on May 4, many stepped up to submit to mailing lists. See bottom of marketing worksheet above for the latest

**Comment by ChrisTheDBA** [comment]
I've done all mine.
